web-ui/src/pages/ProfilePage.jsx

```diff
@@ -16,6 +16,7 @@ import EarnedCertificationsTable from '../components/certifications/EarnedCertif
 import Profile from '../components/profile/Profile';
 import SkillSection from '../components/skills/SkillSection';
 import ProgressBar from '../components/contribution_hours/ProgressBar';
+import VolunteerTables from '../components/volunteer/VolunteerTables';
```

Comment:
This page is accessed by clicking the avatar of the current user in the upper right.